### PR TITLE
feature(extension): Complete dark theme for the browser extension

### PR DESCRIPTION
**Pull request by drjoho**
- #1444 

### Additional information can be found here: 
- 

### Testing reference: 
- 

### I have conducted the following for this PR: 
- [x] I validated this code in Chrome and FF 
- [x] I validated this fix in my local env
- [ ] I provided details for testing
- [ ] This PR has been reviewed and is ready for test  
- [ ] I understand that the title of this PR will be used for the next release notes.

### CODE REVIEW

File: accessibility-checker-extension/src/ts/devtools/components/reportSection.tsx [hunk 1]
```diff
@@ -287,8 +287,8 @@ export class ReportSection extends React.Component<ReportSectionProps, ReportSec
                         inline={true}
                         onClick={() => {
                             let appController = getDevtoolsAppController();
-                            appController.setSecondaryView("summary");
-                            appController.openSecondary("totalIssuesCount");
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/devtools/components/reportSection.tsx:291]: Something doesn't look right here

Comment (drjoho) [reply accessibility-checker-extension/src/ts/devtools/components/reportSection.tsx:291]: Fixed somehow the appController.setSecondaryView before ("summary") got deleted. It is back now.

Comment (drjoho) [reply accessibility-checker-extension/src/ts/devtools/components/reportSection.tsx:291]: Also fixed the Mask group issues.

Comment (drjoho) [reply accessibility-checker-extension/src/ts/devtools/components/reportSection.tsx:291]: All ready now except for the bee icon for the popup. This is in the manifest.json file and I don't know how to do a browser detection in the manifest file.

**Comment by ErickRenteria** [comment]
Validated

**Review by ErickRenteria — APPROVED** [review]

**Pull request merged**